src/plugins/mpi/pmix/pmixp_agent.c

```diff
@@ -291,6 +295,65 @@ static void *_pmix_timer_thread(void *unused)
 	return NULL;
 }
 
+/*
+ * thread for codes from abort
+ */
+static void *_pmix_abort_thread(void *args)
+{
+	PMIXP_DEBUG("Start abort thread");
+
```

Comment:
Add `_abort_handle` next to `_io_handle` (https://github.com/BorisBochkaryov/slurm/blob/master/src/plugins/mpi/pmix/pmixp_agent.c#L56)